ios/Classes/ApproovHttpClientPlugin.m

```diff
@@ -471,8 +471,37 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     } else if ([@"setDevKey" isEqualToString:call.method]) {
         [Approov setDevKey:call.arguments[@"devKey"]];
         result(nil);
-    } else if ([@"getMessageSignature" isEqualToString:call.method]) {
+  } else if ([@"getMessageSignature" isEqualToString:call.method]) {
```

Comment:
Looks like the indentation goes off at this point for all the new code.  (please fix before merge)